test/runtests.jl

```diff
@@ -10,6 +23,28 @@ function gradtest(model, input)
     return true
 end
 
+function normalize(data)
+    cmean = reshape(Float32[0.485, 0.456, 0.406],(1,1,3,1))
+    cstd = reshape(Float32[0.229, 0.224, 0.225],(1,1,3,1))
+    return (data .- cmean) ./ cstd
+end
+
+# test image
+const TEST_PATH = download("https://cdn.pixabay.com/photo/2015/05/07/11/02/guitar-756326_960_720.jpg")
+const TEST_IMG = imresize(Images.load(TEST_PATH), (224, 224))
+# CHW -> WHC
+const TEST_X = permutedims(convert(Array{Float32}, channelview(TEST_IMG)), (3,2,1)) |> normalize
+
+# image net labels
+const TEST_LBLS = readlines(download("https://raw.githubusercontent.com/pytorch/hub/master/imagenet_classes.txt"))
+
+function acctest(model)
+    ypred = model(TEST_X) |> vec
+    top5 = TEST_LBLS[sortperm(ypred; rev = true)]
+
+    return "acoustic guitar" ∈ top5
+end
```

Comment:
In future PRs we could improve this test by evaluating accuracy on a subsample of the testset, maybe 1k images or so, a bit noisy but possibly manageable by CI

Reply:
Agreed or on a subset of "imagenette"? I mostly wanted to avoid trying to download the dataset on CI.

Reply:
yes imagenette is an option, or maybe one can download only the test dataset

Reply:
Like you said, let's do that in another PR?

Reply:
Let's go than